cmd/syncthing/cli/await_insync.go

```diff
@@ -0,0 +1,98 @@
+// Copyright (C) 2021 The Syncthing Authors.
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+package cli
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"os"
+	"time"
+
+	"github.com/urfave/cli"
+)
+
+var awaitInsyncCommand = cli.Command{
+	Name:     "await-insync",
+	HideHelp: true,
+	Usage:    "Block until the current dir finishes syncing",
+	Action:   expects(0, isyncAction),
+}
+
+func isyncAction(c *cli.Context) error {
```

Comment:
isync, seems like it should be inSyncAction?

Reply:
thx